### PR TITLE
Added flye assembly workflow

### PR DESCRIPTION
**Pull request by AnnaSyme**
Added a new folder, and then also a subfolder in here, as I will add some more.

### CODE REVIEW

File: workflows/genome-assembly/assembly-with-flye/Genome-assembly-with-Flye-tests.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    Flye assembly on input dataset(s) (assembly_graph):
+      path: test-data/Flye assembly on input dataset(s) (assembly_graph).graph_dot
+      compare: sim_size
+      delta_frac: 0.2    
+    Flye assembly on input dataset(s) (Graphical Fragment Assembly):
+      path: test-data/Flye assembly on input dataset(s) (Graphical Fragment Assembly).txt
+      compare: sim_size
+      delta_frac: 0.2    
+    Flye assembly on input dataset(s) (assembly_info):
+      path: test-data/Flye assembly on input dataset(s) (assembly_info).tabular
```
Comment (bgruening) [inline workflows/genome-assembly/assembly-with-flye/Genome-assembly-with-Flye-tests.yml:25]: I think it's better to quote those path or avoid whitespaces

Comment (AnnaSyme) [reply workflows/genome-assembly/assembly-with-flye/Genome-assembly-with-Flye-tests.yml:25]: will fix, thanks

**Comment by mvdbeek** [comment]
@AnnaSyme can we do something to help you out with this ?

**Comment by AnnaSyme** [comment]
@mvdbeek do you know why this would be failing at the "combine chunked test results" step?

**Comment by bgruening** [comment]
Bandage container should be available. Lets see what the tests say.

**Comment by mvdbeek** [comment]
> gnuplot: error while loading shared libraries: libGL.so.1: cannot open shared object file: No such file or directory

**Comment by bgruening** [comment]
You are using Singularity images from depot? I have seen the error and syncing building the Singularity images. But I guess Nate needs to sync the CVMFS? I can not trigger that part.

https://github.com/BioContainers/singularity-build-bot/actions/runs/6780748920/job/18433157940

**Comment by mvdbeek** [comment]
No, this is a broken docker images. No singularity

**Comment by mvdbeek** [comment]
Ah, I guess that wasn't clear, it's not bandage

**Comment by bgruening** [comment]
Which container is it? I fixed bandage and the container should work now.

**Comment by mvdbeek** [comment]
https://github.com/BioContainers/multi-package-containers/pull/2813 also did the trick, we merged Delphine's workflow last week that also used bandage.

https://github.com/mvdbeek/galaxy/blob/520fa4a233145d18e3b2a53f49549ea9e4a65cc0/tools/plotting/bar_chart.xml#L1
which doesn't use any dependency at all. Do you know if we have a modern replacement ? barchart seems to be missing from ggplot2?

**Comment by AnnaSyme** [comment]
barchart probably isn't essential. it was just to show contig sizes.

File: workflows/genome-assembly/assembly-with-flye/Genome-assembly-with-Flye-tests.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      delta_frac: 0.2
+    Flye assembly statistics:
+      path: test-data/Flye_assembly_stats.tabular
+      asserts: 
+        has_text:
+          text: "Scaffold L50"
+    'Bandage Image on input dataset(s): Assembly Graph Image':
+      path: test-data/Bandage_image.jpg
+      compare: sim_size
+      delta_frac: 0.2
```
Comment (mvdbeek) [inline workflows/genome-assembly/assembly-with-flye/Genome-assembly-with-Flye-tests.yml:40]:
```suggestion
```

File: workflows/genome-assembly/assembly-with-flye/Genome-assembly-with-Flye-tests.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  job:
+    Input sequence reads:
+      class: File
+      path: test-data/Input_reads.fastqsanger.gz
+      filetype: fastqsanger.gz
+  outputs:
+    Bar chart showing contig sizes:
+      path: test-data/Bar_chart.png
+      compare: sim_size
+      delta_frac: 0.2
```
Comment (mvdbeek) [inline workflows/genome-assembly/assembly-with-flye/Genome-assembly-with-Flye-tests.yml:11]:
```suggestion
```

File: workflows/genome-assembly/assembly-with-flye/Genome-assembly-with-Flye.ga [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+      "when": null,
+      "workflow_outputs": [
+        {
+          "label": "Bar chart showing contig sizes",
+          "output_name": "out_file1",
+          "uuid": "6d0a4e23-d631-4e37-8930-3d22fc91369b"
+        }
+      ]
+    }
+  },
```
Comment (mvdbeek) [inline workflows/genome-assembly/assembly-with-flye/Genome-assembly-with-Flye.ga:300]:
```suggestion
```

File: workflows/genome-assembly/assembly-with-flye/README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+- Flye assembly output - four files: fasta, gfa for bandage, graph_dot file, assembly info
+- Fasta statistics
+- Bandage image
+- Quast report
+- Bar chart
+
+ 
+
+
+
```
Comment (mvdbeek) [inline workflows/genome-assembly/assembly-with-flye/README.md:40]:
```suggestion
```

**Pull request merged**